ios/RNEndpoint.swift

```diff
@@ -17,23 +17,23 @@ struct RNEndpoint {
         type: String,
         videoTracks: [String: VideoTrack] = [:],
         audioTracks: [String: AudioTrack] = [:],
-        tracksMetadata: [String: Metadata] = [:]
+        tracks: [String: TrackData] = [:]
     ) {
         self.id = id
         self.metadata = metadata
         self.type = type
         self.order = RNEndpoint.endpointCounter
         self.videoTracks = videoTracks
         self.audioTracks = audioTracks
-        self.tracksMetadata = tracksMetadata
+        self.tracks = tracks
```

Comment:
still confusing for me why do we have "videoTracks" "audioTracks" and "tracks" in general